src/c++/CMakeLists.txt

```diff
@@ -59,6 +59,12 @@ FetchContent_Declare(
   GIT_SHALLOW ON
 )
 
+FetchContent_Declare(
+  googletest
+  URL https://github.com/google/googletest/archive/9406a60c7839052e4944ea4dbc8344762a89f9bd.zip
```

Comment:
It looks like `9406a60c7839052e4944ea4dbc8344762a89f9bd` from the URL is a commit sha. But is it pertaining to a particular released version? I think it would be good if we stick to a released version (e.g. something in the [tags](https://github.com/google/googletest/tags)). You can still use a commit sha .zip URL, but adding a comment to the CMake to say what tagged version it is would be helpful for people reading this and understanding what dependencies PA has.

Reply:
Ah, gotcha. I see in the release notes that they recommend building with the latest commit--sounds good 👍.